metrics/core/src/main/java/org/hyperledger/besu/metrics/prometheus/PrometheusMetricsSystem.java

```diff
@@ -170,7 +181,61 @@ public void createGauge(
     final String metricName = convertToPrometheusName(category, name);
     if (isCategoryEnabled(category)) {
       final Collector collector = new CurrentValueCollector(metricName, help, valueSupplier);
-      addCollectorUnchecked(category, collector);
+      registerCollector(category, collector);
+    }
+  }
+
+  @Override
+  public void trackExternalSummary(
+      final MetricCategory category,
+      final String name,
+      final String help,
+      final Supplier<ExternalSummary> summarySupplier) {
```

Comment:
I don't truly understand why we need Supplier/Collectors in this method, but I'm guessing it has to do with pluggable metrics?